### PR TITLE
Fix KeyError

### PR DESCRIPTION
**Pull request by shahprog**
If a .sxcu file don't have `Arguments`  which causes a KeyError.

### CODE REVIEW

**Comment by naveen521kk** [comment]
Thank you for your PR.

Can you pull the master branch so that it runs test? Also, is it possible to add a small test? If not possible, I will do ;-)

File: sxcu/sxcu.py [hunk 1]
```diff
@@ -68,7 +68,8 @@ def __init__(
                 con = json.load(sxcu_file)
             # requests url already contain `/upload` removing that.
             self.subdomain = "/".join(con["RequestURL"].split("/")[:-1])
-            self.upload_token = con["Arguments"]["token"]
+            if "Arguments" in con.keys():
```
Comment (naveen521kk) [inline sxcu/sxcu.py:71]: This is lot better.
```suggestion
            if "Arguments" in con:
```

**Review by naveen521kk — CHANGES_REQUESTED** [review]

**Review by naveen521kk — APPROVED** [review]
LGTM! I will do a follow up on tests.

**Comment by shahprog** [comment]
Also it has some more issue. I was making an GUI for uploading jn sxcu and the lib was doing some problem. Then i need to fix them in the source to make that GUI executable work. Some code in the __init__ ar causing the tkinter GUI to fail run the script.

**Comment by naveen521kk** [comment]
Do you mean this
https://github.com/naveen521kk/sxcu/blob/master/sxcu/__init__.py#L11-L18

> Some code in the init are causing the tkinter GUI to fail run the script.

Could you exactly say what is the issue?

**Comment by shahprog** [comment]
```py
 try: 
     from importlib.metadata import version as importlib_metadata_version 
 except ImportError: 
     from importlib_metadata import version as importlib_metadata_version 
  
 __version__ = importlib_metadata_version("sxcu") 
```

Removing these line solved the issue Somehow it caused an error when I use pyinstaller to make a onefile

**Comment by naveen521kk** [comment]
It doesn't hurt for me to remove it. What it does gets the version and sets to the variable. It can also be hard coded. I will be happy if you do a PR.

**Comment by naveen521kk** [comment]
@shahprog I removed it in https://github.com/naveen521kk/sxcu/commit/447b2d9296ff6c75c07836c4f545b44d96a81748
I will do a release tomorrow and then you can use it.

**Comment by naveen521kk** [comment]
@all-contributors please add @shahprog for bug, code

**Pull request merged**